tutorials/getting-started/getting-started.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Getting Started
+layout: tutorial
+---
+
+# Getting Started
+
+In this guide, we’ll cover quickly installing Apostrophe locally for testing and development and testing. First, you’ll need to get the various dependencies installed:
+
+1. Homebrew (Linux and MacOS)
```

Comment:
Linux does not use Homebrew. That's MacOS only. On Linux developers would use the package manager of the Linux distribution they chose.